packages/apputils/src/sessioncontext.tsx

```diff
@@ -1418,6 +1423,7 @@ namespace Private {
   export function populateKernelSelect(
     node: HTMLSelectElement,
     options: SessionContext.IKernelSearch,
+    currentKernelDisplayName: string,
```

Comment:
Thanks for woking on this @gereleth 🚀 !

Perhaps we could use a default value so that we do not change the signature of the method for anyone else using it?

Thanks!